docs/guides/social-signin/microsoft.mdx

```diff
@@ -123,23 +124,23 @@ Follow these steps to add Microsoft as a social sign-in provider to your Ory Clo
 3. Encode the Jsonnet snippet with [Base64](https://www.base64encode.org/) or host it under an URL accessible to Ory Cloud.
 4. Download the Identity Service config from your Ory Cloud project and save it to a file:
 
-  ```shell
-  ## List all available projects
-  ory list projects
+```shell
+## List all available projects
+ory list projects
 
-  ## Get config
-  ory get identity-config <project-id> --format yaml > identity-config.yaml
-  ```
+## Get config
+ory get identity-config <project-id> --format yaml > identity-config.yaml
+```
 
 5. Add the social sign-in provider configuration to the downloaded config. Add the Jsonnet snippet with mappings as a Base64
    string or provide an URL to the file.
 
-  ```yaml
-  selfservice:
-    methods:
-      oidc:
-        config:
-          providers:
+```yaml
```

Comment:
The formatting tool removes the indents I make to enforce proper formatting for lists. Is there any way we could stop that and still have the line break guard active? See the screenshot for a visual comparison.

If there's no way to do that then please exclude the social sign in docs from this pr. I just spent some time fixing the formatting in these docs and this change basically screws most of that up lol

<img width="2034" alt="image" src="https://user-images.githubusercontent.com/29116492/171620420-338c5bc3-7312-41da-8aed-a8071a44bca1.png">

Reply:
In yaml
```yaml
foo:
- bar
- baz
```
and
```yaml
foo:
  - bar
  - baz
```
is the same? Why would we have to use your indentation? In my opinion the variant enforced by prettier even makes more sense, because the array is under the key.

Reply:
Thank you for paying close attention to this! :) 

In YAML you can write lists like the following and they are all equivalent.

```
foo:
  - bar
  - baz
```

```
# this one you see the most often
foo:
- bar
- baz
```

```
foo: ['bar', 'baz']
```

Hope this helps